src/lib.rs

```diff
@@ -239,6 +310,40 @@ mod test {
     use super::*;
     use std::str::FromStr;
 
+    #[test]
+    #[cfg(feature = "serde")]
+    fn json_ser() {
+        fn test_json_ser(val: u64) {
+            let gas = NearGas::from_gas(val);
+            let ser = serde_json::to_string(&gas).unwrap();
+            assert_eq!(ser, format!("\"{}\"", val));
+            let de: NearGas = serde_json::from_str(&ser).unwrap();
+            assert_eq!(de.as_gas(), val);
+        }
+
+        test_json_ser(u64::MAX);
+        test_json_ser(8);
+        test_json_ser(0);
+    }
+
+    #[test]
+    #[cfg(feature = "serde")]
+    fn borsh() {
+        fn test_borsh_ser(val: u64, raw: &str) {
```

Comment:
Use a slice of bytes instead of a debug string representation:

```suggestion
        fn test_borsh_ser(val: u64, expected_serialized_value: &[u8]) {
```